runtime/src/prioritization_fee.rs

```diff
@@ -29,6 +41,24 @@ impl PrioritizationFeeMetrics {
         saturating_add_assign!(self.total_update_elapsed_us, val);
     }
 
+    fn update_prioritization_fee(&mut self, fee: u64) {
+        if fee == 0 {
+            saturating_add_assign!(self.non_prioritized_transactions_count, 1);
+            return;
+        }
+
+        // update prioritized transaction fee metrics.
+        saturating_add_assign!(self.prioritized_transactions_count, 1);
+
+        if fee > self.max_prioritization_fee {
+            self.max_prioritization_fee = fee;
+        }
```

Comment:
might be a little cleaner as...

```suggestion
        self.max_prioritization_fee =
            self.max_prioritization_fee.max(fee);
```